lib/src/api/context.dart

```diff
@@ -0,0 +1,4 @@
+import 'package:mobx/src/core/context.dart';
+
+/// Global Mobx context.
+final ReactiveContext mobxContext = ReactiveContext();
```

Comment:
How about calling it `globalContext` or `defaultContext`?

Reply:
I thought about this a little more, and if we want to explore the Zone thing at some point, maybe "currentContext" or "activeContext" would be a decent name? That way we can replace it with a getter if needed.

Reply:
Yup that sounds good

Reply:
let's go with `currentContext`